k8s/celery-beat/deployment.yml

```
@@ -0,0 +1,79 @@
+      app: celery-beat
+  template:
+    metadata:
+      labels:
+        app: celery-beat
+    spec:
+      containers:
+      - name: celery-beat
+        image: ghcr.io/khulnasoft/reconpoint/celery-beat:latest
+        command: ["celery", "-A", "reconPoint", "beat", "-l", "INFO", "--scheduler", "django_celery_beat.schedulers:DatabaseScheduler"]
+        env:
+          - name: CELERY_BROKER
```

yamllint (1.35.1)

[warning] wrong indentation: expected 8 but found 10

(indentation)